### ISSUE TITLE
fingerprint displayed for the wrong XMPP resource, marked as verified

### ISSUE THREAD

**Issue by WillMorrison** [guardianproject/ChatSecureAndroid#604]
I am signed in with a google account to chatsecure. I open a chat to a contact using pidgin, verify their fingerprint via question, and they verify mine. Then we end the chat, they sign out of pidgin, and sign into chatsecure on a tablet, and open a new chat with me. They verify my fingerprint via question, but my chatsecure app says their fingerprint is already verified. View profile on my chatsecure shows their pidgin fingerprint and XMPP resource, but the chat session is definitely between two chatsecure apps, and I never verified their fingerprint.

To recap, I didn't verify their chatsecure fingerprint, but it shows me their pidgin fingerprint and says the chat is authenticated.

Android chatsecure v14.0.9

**Comment by n8fr8** [comment]
Do you still have this problem with the latest v14.1.1 release?
